### ISSUE TITLE
Viewport meta tag

### ISSUE THREAD

**Issue by soniktrooth** [kalamuna/kalastatic#37]
We're missing a viewport meta tag for responsiveness to kick in on small devices. Anyone have strong opinions?
```
<meta name="viewport" content="width=device-width, initial-scale=1">
```
or 
```
<meta name="viewport" content="width=device-width">
```

http://css-tricks.com/snippets/html/responsive-meta-tag/

**Comment by madeofpeople** [comment]
Pull request welcome!
Any pros and cons to either of your approaches?

**Comment by soniktrooth** [comment]
This is solved by #71
